dojo/api_v2/views.py

```diff
@@ -2482,50 +3528,56 @@ def report_generate(request, obj, options):
     test = None
     endpoint = None
     endpoints = None
-    endpoint_all_findings = None
     endpoint_monthly_counts = None
-    endpoint_active_findings = None
-    accepted_findings = None
-    open_findings = None
-    closed_findings = None
-    verified_findings = None
-    report_title = None
-    report_subtitle = None
 
     include_finding_notes = False
     include_finding_images = False
     include_executive_summary = False
     include_table_of_contents = False
 
     report_info = "Generated By %s on %s" % (
-        user.get_full_name(), (timezone.now().strftime("%m/%d/%Y %I:%M%p %Z")))
+        user.get_full_name(),
+        (timezone.now().strftime("%m/%d/%Y %I:%M%p %Z")),
+    )
 
     # generate = "_generate" in request.GET
     report_name = str(obj)
-    report_type = type(obj).__name__
+    type(obj).__name__
 
-    include_finding_notes = options.get('include_finding_notes', False)
-    include_finding_images = options.get('include_finding_images', False)
-    include_executive_summary = options.get('include_executive_summary', False)
-    include_table_of_contents = options.get('include_table_of_contents', False)
+    include_finding_notes = options.get("include_finding_notes", False)
+    include_finding_images = options.get("include_finding_images", False)
+    include_executive_summary = options.get("include_executive_summary", False)
+    include_table_of_contents = options.get("include_table_of_contents", False)
 
     if type(obj).__name__ == "Product_Type":
         product_type = obj
 
         report_name = "Product Type Report: " + str(product_type)
-        report_title = "Product Type Report"
-        report_subtitle = str(product_type)
-
-        findings = ReportFindingFilter(request.GET, prod_type=product_type, queryset=prefetch_related_findings_for_report(Finding.objects.filter(
-            test__engagement__product__prod_type=product_type)))
-        products = Product.objects.filter(prod_type=product_type,
-                                          engagement__test__finding__in=findings.qs).distinct()
-        engagements = Engagement.objects.filter(product__prod_type=product_type,
-                                                test__finding__in=findings.qs).distinct()
-        tests = Test.objects.filter(engagement__product__prod_type=product_type,
-                                    finding__in=findings.qs).distinct()
+        str(product_type)
```

Comment:
```suggestion

```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/commit/09e8746cdb2cce9fd1e93630862bc819b95e5c15